solr/modules/ltr/src/java/org/apache/solr/ltr/store/rest/ManagedFeatureStore.java

```diff
@@ -92,6 +93,14 @@ public synchronized FeatureStore getFeatureStore(String name) {
     return stores.get(name);
   }
 
+  public synchronized boolean isMissingFeatureStore(String name) {
```

Comment:
Yes, I had already noticed that. But I decided to create this method (and use also at line 173) because the 'stores' variable is private and I need to do the contain in the 'LTRFeatureLoggerTransformerFactory' class.

Reply:
I will try to replace it with a getter